### PR TITLE
Guide on how to store secrets locally

### PR DESCRIPTION
**Pull request by ge0ffrey**
Handy guide on how to deal with local secrets, SEO friendly.

### CODE REVIEW

File: docs/src/main/asciidoc/config.adoc
Comment (gastaldi) [inline docs/src/main/asciidoc/config.adoc:155]: This is an optional step, given that Quarkus already registers the `.env` file as a ConfigSource automatically. You can also use lowercase keys in the `.env` file

File: docs/src/main/asciidoc/config.adoc
Comment (sberyozkin) [inline docs/src/main/asciidoc/config.adoc:134]:
```suggestion
== Store secrets in environment properties file
```

I thought you were referring to `application.properties` :-)

File: docs/src/main/asciidoc/config.adoc
Comment (sberyozkin) [inline docs/src/main/asciidoc/config.adoc:137]:
```suggestion
for security reasons. One way is to store them in a local environment properties (`.env`) file:
```

**Comment by sberyozkin** [comment]
Hi @ge0ffrey Thanks for the PR, not sure my suggestions make sense above, but one thing I'd like to ask to to add a small subsection to the end of https://quarkus.io/guides/security-overview#more-about-security-features-in-quarkus, as there is a valid concern some security related content is hidden if one starts for example from the Quarkus Security Overview doc

File: docs/src/main/asciidoc/config.adoc
Comment (ge0ffrey) [reply docs/src/main/asciidoc/config.adoc:155]: Do you mean the `foo.api-key` entry isn't needed for it to work?

Comment (ge0ffrey) [reply docs/src/main/asciidoc/config.adoc:155]: Fixing like this:
image

**Comment by ge0ffrey** [comment]
@sberyozkin Fixed.

Ready to review again :)

**Review by gastaldi — APPROVED** [review]

**Review by sberyozkin — APPROVED** [review]

**Comment by sberyozkin** [comment]
Thanks @ge0ffrey, would you like to squash commits before we merge? Thanks

**Comment by ge0ffrey** [comment]
Isn't the squash&merge button faster?

Happy to take a look at it when I have some time (on phone now).

**Comment by gastaldi** [comment]
The `Squash and Merge` button is disabled in the Quarkus repository, squashing needs to be done in the PR.

**Comment by ge0ffrey** [comment]
@gastaldi @sberyozkin Squashed, I hope that helps :)

Great project!

**Pull request merged**
